js/mirador_viewer.js

```diff
@@ -12,10 +12,13 @@
      */
     Drupal.behaviors.Mirador = {
         attach: function (context, settings) {
+            settings.mirador.instances = {}
```

Comment:
Untested, but thinking something like:

```suggestion
            Drupal.IslandoraMirador.instances = Drupal.IslandoraMirador.instances  || {}
```